### ISSUE TITLE
[Feature Request]:DateTimePicker hour minutes seconds AM/PM add screen reader instructions

### ISSUE THREAD

**Issue by lormanlau** [SAP/ui5-webcomponents#7980]
### Feature Request Description

Currently right now the DateTimePicker does not announce instructions on how to navigate for the time in picker popup. A11y team is asking for the hours, minutes, seconds, AM/PM to have some directions. Screen reader does not announces information about role of element, it is an interactive element that user can select values and keyboard handling keys to use to navigate through element. No clear information is provided for non-sighted users to interact with element

### Proposed Solution

Add screen reader instructions such as "To navigate use the up and down arrow keys" 
Reference: https://sapui5.hana.ondemand.com/#/entity/sap.m.DateTimePicker/sample/sap.m.sample.DateTimePicker

### Organization

SAP SuccessFactors

### Priority

Medium

### Privacy Policy

- [X] I’m not disclosing any internal or sensitive information.

**Comment by petyabegovska** [comment]
Hello @SAP/ui5-webcomponents-topic-b, please check this ACC request.
